engine/runtime-integration-tests/src/test/java/org/enso/interpreter/test/DiagnosticFormatterTest.java

```diff
@@ -15,18 +15,29 @@
 import org.enso.interpreter.runtime.util.DiagnosticFormatter;
 import org.enso.test.utils.ContextUtils;
 import org.graalvm.polyglot.PolyglotException;
+import org.junit.AfterClass;
 import org.junit.Before;
+import org.junit.BeforeClass;
 import org.junit.ClassRule;
 import org.junit.Test;
 
 public class DiagnosticFormatterTest {
   @ClassRule
   public static final ContextUtils ctxRule =
       ContextUtils.newBuilder()
-          .withModifiedContext(
-              b -> b.option(RuntimeOptions.STRICT_ERRORS, "true").environment("NO_COLOR", "true"))
+          .withModifiedContext(b -> b.option(RuntimeOptions.STRICT_ERRORS, "true"))
           .build();
 
+  @BeforeClass
+  public static void before() {
+    System.setProperty("NO_COLOR", "true");
```

Comment:
Setting and clearing a system property in `@BeforeClass` & co. seems like the last _level of rescue_. This is just a single test that runs in the same JVM. What if some other test initializes the system that reads `NO_COLOR` before?